assets/prometheus-k8s/rules.yaml

```diff
@@ -388,6 +388,13 @@ spec:
       for: 30m
       labels:
         severity: warning
+    - alert: AlertmanagerReceiversNotConfigured
+      annotations:
+        message: Alerts are not configured to be sent to a notification system. Configuring
+          an alert route will allow you to be notified when important failures occur.
+      expr: cluster:alertmanager_routing_enabled:max > 0
+      labels:
+        severity: warning
```

Comment:
From what I understand we want the alert to show up to users in the main openshift console? Do warning alerts also show up, I thought only critical ones do.

Reply:
I am not sure. `warning` level was requested from @smarterclayton. Initially this had `critical` level.

Reply:
Lets follow up with @smarterclayton to see what he meant.

Reply:
Verified myself. Alerts with both severities are included in dashboard.
Here is an example (`ServiceCatalogControllerManagerEnabled` has a `warning` severity)

![Screenshot_20191219_162018](https://user-images.githubusercontent.com/3531758/71185349-925b4c00-227b-11ea-88b1-ef61147946e3.png)

Reply:
Nice! no, sgtm